README.md

```diff
@@ -31,7 +33,7 @@ admission control can be delegated to controllers such as [gatekeeper](https://g
 - ✔️ Up-to-date [documentation](/docs).
 - ✔️ Monitoring via [metrics](/docs/reference/metrics.md)
 - ✔️ API validation and defaulting.
-- ⏳ Scalability tests in progress.
+- ✔ Scalability tests.
```

Comment:
What are the numbers? this is not just about having the tests, but more about having reasonable scaling limits.

Reply:
I think we should have a test report about scalability, and we can refer to the result here.

Reply:
@mwielgus found it hard to give numbers given that in small clusters most of the load is dominated by the kubelets.

Maybe we can say: `Reproducible load tests`, with a link to the readme.